components/00-base/01-colors/colors.twig

```diff
@@ -5,23 +5,28 @@
     heading_level: 2,
     heading: colors_heading,
   } %}
-
-  {% for palette in palettes %}
-    {% set palette_machine = palette.palette_heading|lower %}
-    {% include "@atoms/text/headings/_heading.twig" with {
-      heading_level: 3,
-      heading: palette.palette_heading,
-    } %}
-    <ul {{ bem('list', [palette_machine], colors_base_class) }} data-theme="{{ palette_machine }}">
-      {% for color in palette.colors %}
-        {% set color_machine = color.name|lower|replace({' ': '-'}) %}
-        {% if color.color_reverse == TRUE %}
-          {% set colors_title_modifiers = ['reverse'] %}
-        {% endif %}
-        <li {{ bem("item", [palette_machine ~ '-' ~ color_machine], colors_base_class) }}>
-          <span {{ bem("title", colors_title_modifiers, colors_base_class) }}>{{ color.name }}</span> <br/>
-        </li>
-      {% endfor %}
-    </ul>
-  {% endfor %}
+  <ul>
+    {% for palette in palettes %}
+      {% set palette_machine = palette.palette_heading|lower %}
+      {% include "@atoms/text/headings/_heading.twig" with {
+        heading_level: 3,
+        heading: palette.palette_heading,
+      } %}
+      <ul {{ bem('list', [palette_machine], colors_base_class) }} data-theme="{{ palette_machine }}">
```

Comment:
@josue2591 I think you need to add an `li` and inside it add the `ul` in order to pass [accessibility](https://www.w3.org/WAI/tutorials/page-structure/content/#nested-lists). You might also consider change the root `ul` (line 8) by `ol`

Reply:
Thanks! Changes applied